.github/workflows/codeql-analysis.yml

```diff
@@ -40,9 +46,21 @@ jobs:
       uses: actions/setup-dotnet@v2.1.0
       with:
         dotnet-version: 6.0.301
+        
+    - name: Setup .NET 7.0.x
+      uses: actions/setup-dotnet@v2.1.0
+      with:
+        dotnet-version: 7.0.x
+
+   # Install workload
+    - name: Setup wasm-tools
+      run: dotnet workload install wasm-tools
 
     - name: Build with .NET 6
-      run: dotnet build Microsoft.Identity.Web.sln -f net6.0 -p:FROM_GITHUB_ACTION=true --configuration Release
+      run: dotnet test Microsoft.Identity.Web.sln -f net6.0 -p:FROM_GITHUB_ACTION=true --configuration Release --filter "(FullyQualifiedName!~Microsoft.Identity.Web.Test.Integration)&(FullyQualifiedName!~WebAppUiTests)&(FullyQualifiedName!~IntegrationTests)"
```

Comment:
Do we need a dotnet test? I would think that dotnet build would be enough for Code QL?

Reply:
It's not working with dotnet test. Can revisit later, but at least it's runnign now.